specification/ai/Face/models.session.tsp

```diff
@@ -263,13 +283,21 @@ model LivenessSessionAuditEntry {
 
   @doc("The server calculated digest for this request. If the client reported digest differs from the server calculated digest, then the message integrity between the client and service has been compromised and the result should not be trusted. For more information, see how to guides on how to leverage this value to secure your end-to-end solution.")
   digest: string;
+
+  @doc("The image ID of the session request.")
+  @added(Versions.v1_2_preview_1)
+  sessionImageId?: string;
+
+  @doc("The sha256 hash of the verify-image in the request.")
+  @added(Versions.v1_2_preview_1)
+  verifyImageHash?: string;
 }
 
 @doc("Request of liveness with verify session creation.")
-model CreateLivenessWithVerifySessionContent {
+model CreateLivenessWithVerifySessionMultipartBody {
```

Comment:
This appears to be an SDK break.

Java CI fails due to this (likely a conflict of generated code and customization code).
https://github.com/Azure/azure-rest-api-specs/pull/29781/checks?check_run_id=27836423122

Reply:
It is necessary? AFAIK, the model renaming cannot be modelled via tsp versioning. Hence it appears in v1.1-preview.1 Swagger as well.

Reply:
It's intended change to differentiate the input model. 
I'm open to other solutions, but it's a little complicated.
In v1.1, the operations look like
```
createLivenessSession(CreateLivenessSessionContent)
createLivenessWithVerifySession(CreateLivenessSessionContent)
createLivenessWithVerifySessionWithVerifyImage(CreateLivenessWithVerifySessionContent)

model CreateLivenessWithVerifySessionContent 
{
Parameters: CreateLivenessSessionContent;
VerifyImage: bytes;
}
```
In v1.2, liveness with verify session introduced some specific field, so it can not share the model with liveness session. `CreateLivenessWithVerifySessionContent` is also occupied by multipart request, that's why I renamed all of them to avoid confusion.
Please also note that `CreateLivenessWithVerifySessionRequest` is an additional layer added by SDK folks after our previous release, I'm not sure what's the purpose.
```
createLivenessSession(CreateLivenessSessionParameters)
createLivenessWithVerifySession(CreateLivenessWithVerifySessionParameters)
createLivenessWithVerifySessionWithVerifyImage(CreateLivenessWithVerifySessionRequest)

model CreateLivenessWithVerifySessionMultipartBody 
{
Parameters: CreateLivenessWithVerifySessionParameters;
VerifyImage: bytes;
}
```

Reply:
The `CreateLivenessWithVerifySessionRequest` was due to https://github.com/Azure/azure-rest-api-specs/pull/29723/files for the spread thing (without it and the `@body` under it, the model `CreateLivenessWithVerifySessionContent` would be spread in that API).

Reply:
I see. It is `CreateLivenessSessionContent` now renamed to `CreateLivenessSessionParameters` (and 2 added properties).

And `createLivenessWithVerifySession`/`createLivenessWithVerifySessionWithVerifyImage` API now need a different model, hence `CreateLivenessWithVerifySessionParameters` (and further 2 more properties).

The versioning of these models is hard, when rename and add is mixed together. I don't have a good recommendation either. As both version is preview, maybe we can just make the break.

Reply:
I'm thinking about another option. Redefine the `CreateLivenessWithVerifySessionContent` model. 
```
createLivenessSession(CreateLivenessSessionContent)  
createLivenessWithVerifySession(CreateLivenessWithVerifySessionContent)  
createLivenessWithVerifySessionWithVerifyImage(CreateLivenessWithVerifySessionMultipartContent) 
```
In this way we can avoid breaking changes on `createLivenessSession` since it's model does not change.
However, the breaking changes on `CreateLivenessWithVerifySessionContent` would looks weird since it was used for multipart in v1.1.
On the other hand, `CreateLivenessWithVerifySessionContent` was not exposed in .NET/Java since we had a layered of customization to merge `createLivenessWithVerifySession` and `createLivenessWithVerifySessionWithVerifyImage`.
This way would lower the breaking change scope than current implementation. 
Let me discuss it with SDK developers in our team.